manifests/init.pp

```diff
@@ -61,7 +61,27 @@
   Optional[Integer] $logconfig_file_max_log_size = undef,
   Optional[Integer] $logconfig_file_backups      = undef,
   Optional[String] $logconfig_file_category      = undef,
+
+  # Padoptions
+  Etherpad::Stpadoptions $padoptions = {},
+
 ) {
+  #Default parameters for padoptions
```

Comment:
I'm not sure this comment adds a lot of value.

Reply:
Fixed.